### PR TITLE
Feature 4 test client imei parsing

### LINKED ISSUE #4: Test client IMEI parsing
As a developer I'd like to have first unit tests written for the application.

### PR DESCRIPTION
**Pull request by villejuutila**
Resolves #4

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -67,7 +85,8 @@ async fn handle_valid_connection(
     socket_address: SocketAddr,
     imei: Option<String>
 ) -> Result<(), Box<dyn Error>> {
-    let mut file = OpenOptions::new().read(true).create(true).append(true).open(format!("{}.txt", imei.unwrap())).expect("Failed to open file");
+    // Uncomment line below to enable writing to file
+    // let mut file = OpenOptions::new().read(true).create(true).append(true).open(format!("{}.txt", imei.unwrap())).expect("Failed to open file");
```
Comment (anttileppa) [inline src/main.rs:89]: Exclude this from tests, instead of commeting it out

File: src/main.rs [hunk 2]
```diff
@@ -81,17 +100,18 @@ async fn handle_valid_connection(
...
+            debug!("Received ping from client {}", socket_address);
             continue;
         }
         let (_, frame) = parser::tcp_frame(&buffer).expect("Failed to parse TCP frame");
         let amount_of_records = frame.records.len();
-        println!("Received {} records from client {}", amount_of_records, socket_address);
-        writeln!(file, "{:#?}", frame).unwrap();
+        debug!("Received {} records from client {}", amount_of_records, socket_address);
+        // Uncomment line below to enable writing to file
+        // writeln!(file, "{:#?}", frame).unwrap();
```
Comment (anttileppa) [inline src/main.rs:110]: Exclude this from tests, instead of commeting it out

**Review by anttileppa — APPROVED** [review]

File: src/main.rs (hunk 1 above)
Comment (villejuutila) [reply src/main.rs]: Resolved.

File: src/main.rs (hunk 2 above)
Comment (villejuutila) [reply src/main.rs]: Resolved.

**Pull request merged**